### ISSUE TITLE
automate setting community flag

### ISSUE THREAD

**Issue by pombase-admin** [pombase/canto#654]
In [curation-tool:#655], we realized that the 'community curatable' flag can be automagically set for any publication that meets these criteria:

- There's a curation session for the paper, assigned to someone; 

AND

- The assignee isn't one of us (i.e. p.name <> 'Antonia Lock' and p.name <> 'Val Wood' and p.name <> 'Midori Harris' and p.name <> 'Antonia Nilsson' and p.name <> 'Kim Rutherford').

Original comment by: mah11

**Comment by ValWood** [comment]
First off the community curation flag should indicate that a paper is community curatable. 
This can be set automatically if a curator is  ever assigned and it is not one of us 3 (i.e I don't need to set this manually in the triage interface).

Then we want a way to evaluate which of the sessions that were sent out for community curation, which of these were "done' by the community. This will be "any session that has the CC flag and at least one annotation from a community curator". This can be added as an admin list.

Of course there are many where the community added  only a few terms and we added lots more. We can evaluate this later when we figure out the ratio curator:community on individual annotations in a session  if we want to evaluate.

(We know that we are assigned as curators of lots of CC sessions currently, that NEVER went out for community curation. We are adjusting these)

**Comment by Antonialock** [comment]
Also we want the flag to change if we change the curator from a community one (who might have ignored our invitation) to one of us,

**Comment by ValWood** [comment]
No, in this case we want to keep the community curatable flag. We want to be able to evaluate how many people ignored us. The eventual curator will be captured by the new list.

**Comment by ValWood** [comment]
Step 3.
I am trying to establish the number of publications identified as “community curatable”. This table is ‘polluted’ by non-curatable papers

Can we remove the "community curatable" flag from every paper which has NOT been classified as "curatable"...I was doing a few manually, but there seem to be quite a lot of method,wrong organism, loaded in error etc) and the list is ~1400 long at present.

I can't remember how the 'community curatable' flag gets set, but I suggest this defaults to not set, and I set to on at triage.

This flag can also be set automatically to "on" for any session which becomes assigned to a community curator after the triage process (for example if a user self selects a paper using the PMID)

A more accurate number will allow us eventually to assess the % of sessions we have assigned which have been completed.
We don't need to do this yet....... We will get much better numbers once all of the active sessions from the recent reminders are through.... and I have weeded out the remaining non CC papers
